payments/mobile-pay/index.md

```diff
@@ -227,8 +227,8 @@ sequenceDiagram
 [mobile-pay]: /payments/mobile-pay/
 [payex-reconciliation-and-settlement]: /payments/mobile-pay/other-features#settlement-and-reconcilliation
 [redirect-implementation]: /payments/mobile-pay/redirect
-[swish-e-commerce]: /payments/swish/#swish-e-commerce-redirect-to-payment-pages
-[swish-m-commerce]: /payments/swish/#swish-m-commerce-redirect-to-payment-pages
+[swish-e-commerce]: /payments/swish#swish-e-commerce-redirect-to-payment-pages
+[swish-m-commerce]: /payments/swish#swish-m-commerce-redirect-to-payment-pages
```

Comment:
Why is Mobile Pay linking to Swish?

Reply:
Excellent question. I've added this as a sub-task in DX-259 because there are plenty of things to fix in Mobile Pay